### DISCUSSION TITLE
Add more bi-direction communication example

### DISCUSSION THREAD

**Discussion by mzy2240** [writer/writer-framework#20] [General]
I think the current example lean to show the capability of building the dashboard from Python, and somewhat overlook the importance of interactivity (from dashboard to Python). IMO, this is the main advantage over other frameworks, and thus should be highlighted.

**Comment by ramedina86** [comment]
Hey, I'm really interested in this. Right now, my priority is promoting Streamsync and refine the positioning of the framework. I'm taking a short break from intense coding to work on these type of things (writing on Medium, Reddit, LinkedIn, etc).

The `hello` app is definitely a key element here. I can also look into creating more apps that could serve different purposes -some sort of "Gallery".

What ideas do you have in mind? How would reflect this in a demo app? Keen to hear.

**Comment by mzy2240** [comment]
Maybe starting from something simple, like

1. creating a text input or text area from script, and then use python to validate the input from the user.
2. use button to trigger chart re-rendering
3. use dropdown menu to switch between different plots
4. use slider to update the data range in a plot
